cmd/limactl/inspect.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/goccy/go-yaml"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newInspectCommand() *cobra.Command {
+	inspectCommand := &cobra.Command{
+		Use:   "inspect [flags] [INSTANCE]...",
+		Short: "Inspect instances of Lima.",
+		Long: `Inspect instances of Lima.
+		Includes both the current state, as well as the combined configuration used to create the instance.
+		The output can be presented in one of several formats, using the --format <format> flag.
+		
+		  --format json - output in json format
+		  --format yaml - output in yaml format
+		  --format '{{ <go template> }}' - if the format begins and ends with '{{ }}', then it is used as a go template.
```

Comment:
Not robust, `{{` and `}}` can appear anywhere in the string

Reply:
e.g.,
https://github.com/lima-vm/lima/blob/f8f3a7b5664b877421d89745ca74f5891e7061ba/examples/docker.yaml#L6

Reply:
Actually, I think the simplest solution is, "if a reserved work like 'json' or 'yaml' or 'table', then use those, else just treat it as a go template".